app/controllers/signup.js

```diff
@@ -223,10 +225,12 @@ export default class SignupController extends Controller {
       if (status === 204) {
         window.open('https://realdevsquad.com/goto', '_self');
       } else {
-        alert('Something went wrong. Please check console errors.');
+        // alert('Something went wrong. Please check console errors.');
       }
```

Comment:
Do we need this else statement if there's nothing anyway in there?

Reply:
Ohhh, my bad, the code needs to be uncommented.